src/utils/useGetAddressFromLatLng.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-underscore-dangle,camelcase */
+import { useEffect, useState } from 'react'
+import { LatLng } from 'leaflet'
+import proj4, { InterfaceCoordinates } from 'proj4'
+import { ENDPOINTS } from '../../example/config'
+import { CRS_CONFIG } from './getCrsRd'
+
+type Links = {
+  self: {
+    href: string
+  }
+  next: {
+    href: string
+  }
+  previous: {
+    href: string
+  }
+}
+
+type VboStatus = { code: string; omschrijving: string }
+
+type Address = {
+  _links: Links
+  _display: string
+  landelijk_id: string
+  hoofdadres: boolean
+  vbo_status: VboStatus
+  dataset: string // Todo: could be an enum?
```

Comment:
Do we want to make this an enum?

Reply:
Yes, but we have to figure out what enum's it can accept. That means we have to dive into the API's, think that's a bit out of scope for now.

Reply:
In that case a string will be fine. But let's remove the TODO then.